core/src/main/java/org/springframework/security/authentication/UsernamePasswordAuthenticationToken.java

```diff
@@ -71,6 +75,11 @@ public UsernamePasswordAuthenticationToken(Object principal, Object credentials,
 		super.setAuthenticated(true); // must use super, as we override
 	}
 
+	public static UsernamePasswordAuthenticationToken authenticated(Object principal, Object credentials,
```

Comment:
For each new public method, please add a JavaDoc that describes what the method does. Please also include a `@since 5.7` attribute.

Reply:
@jzheaux Should I replace:
```
@author Ben Alex
```
at the top with 
```
 @author Norbert Nowak
 @since 5.7
```
or simply add to newly created JavaDoc?

Reply:
Please add `@author Norbert Nowak` to the class-level JavaDoc underneath `@author Ben Alex`.